02-Custom-Login/app/views/home/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="jumbotron text-center">
+  <h2><img class="jumbo-thumbnail" src="https://cdn.auth0.com/styleguide/1.0.0/img/badge.svg"></h2>
+  <h1>RoR Auth0 Sample</h1>
+  <p>Step 2 - Custom Login.</p>
+  <p class="text-center"><%= notice %></p>
+</div>
+
+<div class="container">
+  <div class="col-md-6 col-md-3">
+    <div id="login-container">
+      <%= form_tag(action:'auth') do %>
+          <%= text_field_tag :user, "", class:'form-control' %>
+          <br>
+          <%= password_field_tag :password, "", class:'form-control'%></p>
+          <br>
+          <%= submit_tag("Log In", class: "btn btn-primary") %>
```

Comment:
You used "log in" (2 words) here and "login" (1 word) in line 19. Use a single word everywhere.